priv/templates/boot.eex

```diff
@@ -208,6 +208,15 @@ _replace_os_vars() {
         }1' < "$1" > "$2"
 }
 
+# Private. Run hooks from directory.
+_run_hooks_from_dir() {
+   if [ -d "$1" ]; then
+          for s in $(find "$1" -name "*.sh" -print | sort); do
```

Comment:
Suggestion:

```
# skip symlinks and other oddities
# handle filenames with escapes and oddities
find "$1" -name "*.sh" -type f -print0 | while read -d $'\0' s ; do
```

I would prefer to inline this grep, to grab only files starting with alphanum, skips '.foo.sh' tmpfiles, but it doesn't work on OSX; the alternative that comes to mind involves ugly bashisms

```
# 
find "$1" -name "*.sh" -type f -print0 | grep -0 '^[0-9a-z-A-Z]' | while read -d $'\0' s ; do
```